release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1590,15 +1590,41 @@ Previously, the image pruner stopped when it failed to delete an image. As a res
+
+*	During dependency and upgrade resolution, subscription uniqueness was previously based on the subscribed package name. If two subscriptions in a namespace subscribe to the same package, they are treated as a single subscription internally, resulting in unexpected behaviors. With this fix, subscriptions are now uniquely identified internally within a namespace by `.metadata.name` instead of `.spec.name`. As a result, upgrade and dependency resolution behavior is consistent for namespaces containing multiple `Subscription` objects with the same `.spec.name.` (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932001[*BZ#1932001*])
+
+* When less than one minute remains before an upcoming catalog update polling attempt, the interval jitter function truncates the resync interval down to zero. This caused the Operator Catalog to enter a hot-loop, wasting CPU cycles. This fix increases precision of the jitter function used to calculate resync delays. As a result, the Catalog Operator remains mostly idle until the next catalog update poll.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1932182[*BZ#1932182*])
+
+* During an Operator upgrade, the owner reference of any associated `ServiceAccount` objects was updated to point to the new `ClusterServiceVersion` (CSV) object instead of the old one. This could cause a race condition to occur between the OLM Operator, which reconciles CSVs, and the Catalog Operator, executes install plans, marking the old CSV as `Pending/RequirementsNotMet` due to the service account ownership change. This blocked upgrade completion while the new CSV waited indefinitely for the old CSV to indicate a healthy status. With this fix, instead of updating owner references in one step, the second owner is now appended to any existing owners. As a result, the same service account can satisfy requirements for both the old and the new CSV. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934080[*BZ#1934080*])
+
+*	Cluster service versions (CSV) previously required associated service accounts to either have no `ownerReferences` values set or to have an `ownerReference` value set to the related CSV. This caused `default` service account, which is not created as part of Operator installation, to be unsatisfied as a CSV requirement if its `.metadata.ownerReferences` field was not empty. With this fix, CSVs now require associated service accounts to either have no `ownerReferences` values set to CSVs or to have an `ownerReference` value set to the related. As a result, service accounts with only non-CSV `ownerReferences` values can satisfy the requirements of any CSV. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1935909[*BZ#1935909*])
```

Comment:
s/set to the related/set to the related CSV ??(That seems to be what is in the BZ: _ associated ServiceAccounts to either have no ownerReferences to CSVs or to have an ownerReference to the CSV in question._

I wonder, if the `ownerReferences` parameter should always be plural?  No _an `ownerReference` value_